# HG changeset patch
# User Jaroslav Hajek <highegg@gmail.com>
# Date 1285826677 -7200
#      Thu Sep 30 08:04:37 2010 +0200
# Node ID 4d9f310b32bbec08ba2bb7e662525619d1b4494e
# Parent  4bec51eb58e20e32029e5fb7d571518028846b09
replace Octave_map->octave_map in oct-errno

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,14 @@
+2010-09-30  Jaroslav Hajek  <highegg@gmail.com>
+
+	* oct-errno.cc (octave_errno::list, octave_errno::do_list): Use
+	octave_scalar_map instead of Octave_map.
+	* oct-errno.h: Update decls.
+
 2010-09-30  Jaroslav Hajek  <highegg@gmail.com>
 
 	* pt-idx.cc (tree_index_expression::make_arg_struct): Use octave_map
 	instead of Octave_map.
 	* pt-idx.h: Update decl.
 
 2010-09-29  Jaroslav Hajek  <highegg@gmail.com>
 
diff --git a/src/oct-errno.h b/src/oct-errno.h
--- a/src/oct-errno.h
+++ b/src/oct-errno.h
@@ -40,17 +40,17 @@ protected:
 public:
 
   ~octave_errno (void) { }
 
   static bool instance_ok (void);
 
   static int lookup (const std::string& name);
 
-  static Octave_map list (void);
+  static octave_scalar_map list (void);
 
   static int get (void) { return errno; }
 
   static int set (int val)
   {
     int retval = errno;
     errno = val;
     return retval;
@@ -59,12 +59,12 @@ public:
 private:
 
   std::map<std::string, int> errno_tbl;
 
   static octave_errno *instance;
 
   int do_lookup (const std::string& name);
 
-  Octave_map do_list (void);
+  octave_scalar_map do_list (void);
 };
 
 #endif
